packages/shared/providers.yaml

```diff
@@ -86,6 +92,8 @@ bamboohr:
     token_params:
         grant_type: authorization_code
         request: token
+    proxy:
+        base_url: https://api.bamboohr.com/api/gateway.php/${connectionConfig.params.subdomain}/v1
```

Comment:
params property is no longer needed. Should just be 

connectionConfig.subdomain

csn you update other places in this file too?

thsnks!

Reply:
Thanks for checking this out, apparently this had already existed before. In anyways I have updated this and the rest, thanks for noticing. @khaliqgant